src/test/kotlin/com/igorwojda/integer/fizzbuzz/challenge.kt

```diff
@@ -4,7 +4,19 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun fizzBuzz(n: Int): List<String> {
-    TODO("not implemented")
+    val listFizzBuzz = mutableListOf<String>()
+    for (i in 1..n) {
+        if (i % 15 == 0) {
```

Comment:
Please don't ignore the linter. It already says that cascading ifs should be transformed to a when statement.